app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartController.scala

```diff
@@ -46,6 +47,10 @@ class StartController @Inject()(override val messagesApi: MessagesApi,
   }
 
   val start: Action[AnyContent] = PasscodeAuthenticatedAction { implicit request =>
+    request.getQueryString("continue").foreach { url =>
```

Comment:
Could the continue query parameter be put in the routes file and use the ContinueUrl's provided binding ?
e.g.
GET     /start                          uk.gov.hmrc.agentsubscriptionfrontend.controllers.StartController.start(continue: Option[uk.gov.hmrc.play.binders.ContinueUrl])